pikaur/build.py

```diff
@@ -19,6 +19,21 @@
 )
 
 
+def asroot():
```

Comment:
i think these two guys should go to to `pikaur/core.py` and also rename `asroot` to smth nicer